templates/html/.partials/sidebar.html

```diff
@@ -25,18 +23,71 @@ <h1 class="text-2xl font-thin">{{asyncapi.info.title}} {{asyncapi.info.version}}
       </li>
       {{/if}}
     </ul>
-
     {{#if asyncapi.channels}}
     <h2 class="text-xs uppercase text-grey mt-10 mb-4 font-thin">Operations</h2>
     <ul class="text-sm list-reset mt-2">
+      {{#if params.sidebarOrganization}}
```

Comment:
This is not checking if `params.sidebarOrganization` is `hierarchical` but instead it's just checking `params.sidebarOrganization` is truthy. Also, what about changing `hierarchical` to `byTags`?

```suggestion
      {{#if (equal params.sidebarOrganization 'byTags')}}
```

Reply:
Yeah, my bad. byTags as parameter name would be better suited I think. It's immediately clear to the user what it will do.